mutation-testing/output-pr-mutants/action.yml

```diff
@@ -0,0 +1,153 @@
+name: Output Mutants
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Download artifacts
+      uses: actions/download-artifact@v3
```

Comment:
please use the latest version commit hash.
ex: `uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1`